tests/ui/test_login.py

```diff
@@ -1,23 +1,24 @@
+import os
 import pytest
 
 from pages.desktop.home import Home
 
 
-@pytest.mark.skip(
-    reason='https://github.com/mozilla/addons-server/issues/2462')
-def test_login(base_url, selenium, user):
+@pytest.mark.skipif(os.environ.get('PYTEST_BASE_URL') is None,
+                    reason='Live Server login currently not functioning')
```

Comment:
Correct me if I'm blind, knowing that this is hard to fix, are there any tests currently running that are using the live-server and thus testing the overwritten `live_server` fixture? I can't see any but please correct me if I'm wrong.